{"uid":"dfa344f6bf411508","name":"test_convert","fullName":"tests.test_cli#test_convert","historyId":"f35d72d4a06cff57d75e83eb8762ca60","time":{"start":1668031490643,"stop":1668031510139,"duration":19496},"status":"broken","statusMessage":"PermissionError: [WinError 5] Access is denied: 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\Lib' -> 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\lib'","statusTrace":"Traceback (most recent call last):\n  File \"D:\\a\\conda-build\\conda-build\\tests\\test_cli.py\", line 448, in test_convert\n    main_convert.execute(args)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\cli\\main_convert.py\", line 117, in execute\n    api.convert(f, **args.__dict__)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\api.py\", line 292, in convert\n    return conda_convert(package_file, output_dir=output_dir, show_imports=show_imports,\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 796, in conda_convert\n    convert_from_windows_to_unix(file_path, output_dir, platform,\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 709, in convert_from_windows_to_unix\n    update_lib_contents(directory, temp_dir, 'unix', file_path)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 279, in update_lib_contents\n    src_dir.rename(dst_dir)\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\pathlib.py\", line 1382, in rename\n    self._accessor.rename(self, target)\nPermissionError: [WinError 5] Access is denied: 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\Lib' -> 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\lib'","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1668028756600,"stop":1668028756600,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmp_path","time":{"start":1668031490627,"stop":1668031490643,"duration":16},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"tmpdir","time":{"start":1668031490643,"stop":1668031490643,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_config","time":{"start":1668031490643,"stop":1668031490643,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"testing_workdir","time":{"start":1668031490643,"stop":1668031490643,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"PermissionError: [WinError 5] Access is denied: 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\Lib' -> 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\lib'","statusTrace":"Traceback (most recent call last):\n  File \"D:\\a\\conda-build\\conda-build\\tests\\test_cli.py\", line 448, in test_convert\n    main_convert.execute(args)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\cli\\main_convert.py\", line 117, in execute\n    api.convert(f, **args.__dict__)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\api.py\", line 292, in convert\n    return conda_convert(package_file, output_dir=output_dir, show_imports=show_imports,\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 796, in conda_convert\n    convert_from_windows_to_unix(file_path, output_dir, platform,\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 709, in convert_from_windows_to_unix\n    update_lib_contents(directory, temp_dir, 'unix', file_path)\n  File \"D:\\a\\conda-build\\conda-build\\conda_build\\convert.py\", line 279, in update_lib_contents\n    src_dir.rename(dst_dir)\n  File \"C:\\Users\\runneradmin\\Miniconda3\\lib\\pathlib.py\", line 1382, in rename\n    self._accessor.rename(self, target)\nPermissionError: [WinError 5] Access is denied: 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\Lib' -> 'C:\\\\Users\\\\RUNNER~1\\\\AppData\\\\Local\\\\Temp\\\\tmpu5pfdhnt\\\\lib'","steps":[],"attachments":[{"uid":"fadd4e9fed995239","name":"log","source":"fadd4e9fed995239.txt","type":"text/plain","size":847},{"uid":"25fb1aba21b60536","name":"stdout","source":"25fb1aba21b60536.txt","type":"text/plain","size":61}],"parameters":[],"stepsCount":0,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"testing_workdir::return_to_saved_path","time":{"start":1668031510139,"stop":1668031510139,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_cli"},{"name":"host","value":"fv-az177-7"},{"name":"thread","value":"3764-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_cli"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"dfa344f6bf411508.json","parameterValues":[]}